core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -191,14 +191,8 @@ THE SOFTWARE.
     </j:if>
 
     <div id="page-body" class="app-page-body app-page-body--${layoutType} clear">
-      <j:if test="${layoutType=='two-column'}">
-        <j:set var="mode" value="side-panel" />
-        <d:invokeBody />
```

Comment:
Breaks all context menus.

Reply:
Thanks! Fixed it and added a comment to `side-panel.jelly`